Source/Applications/SystemCenterNotification/Scripts/TSX/Subscriptions/ConfirmPhone.tsx

```diff
@@ -80,61 +80,59 @@ const ConfirmPhone = (props: IProps) => {
-                                </div>
-                                <div className="col-3">
-                                </div>
+            </div>
+            <div className="row" style={{ marginTop: 15 }}>
+                <div className="col">
+                    <form>
+                        <div className="form-row">
+                            <div className="col-4">
                             </div>
-                        </form>
-                    </div>
+                            <div className="col">
+                                <input type="number" max={9} min={0} className="form-control" value={Math.floor(number / 10000)} onChange={(evt) => {
+                                    if (IsInteger(evt.target.value))
+                                        setNumber(x => x + 10000 * (-Math.floor(number / 10000) + parseInt(evt.target.value)))
+                                }} width={100} />
+                            </div>
+                            <div className="col">
+                                <input type="number" max={9} min={0} className="form-control" value={Math.floor(number / 1000) % 10} onChange={(evt) => {
+                                    if (IsInteger(evt.target.value))
+                                        setNumber(x => x + 1000 * (-Math.floor(number / 1000) % 10 + parseInt(evt.target.value)))
+                                }} />
+                            </div>
+                            <div className="col">
+                                <input type="number" max={9} min={0} className="form-control" value={Math.floor((number % 1000) / 100) % 10} onChange={(evt) => {
+                                    if (IsNumber(evt.target.value))
+                                        setNumber(x => x + 100 * (-Math.floor((number % 1000) / 100) % 10 +  parseInt(evt.target.value)))
+                                }} />
+                            </div>
+                            <div className="col">
+                                <input type="number" max={9} min={0} className="form-control" value={Math.floor((number % 100) / 10) % 10} onChange={(evt) => {
+                                    if (IsNumber(evt.target.value))
+                                        setNumber(x => x + 10 * (-Math.floor((number % 100) / 10) % 10 + parseInt(evt.target.value)))
+                                }} />
+                            </div>
+                            <div className="col">
+                                <input type="number" max={9} min={0} className="form-control" value={number % 10} onChange={(evt) => {
+                                    if (IsNumber(evt.target.value))
+                                        setNumber(x => x - (x % 10) + parseInt(evt.target.value))
+                                }} width={100} />
+                            </div>
+                            <div className="col-3">
+                            </div>
```

Comment:
Why do we need this? (There are several empty `div`)
Would be much cleaner to just specify `col-1` on the input ones and center them inside the row via teh appropriate class

Reply:
I'm not sure, I don't think I made changes to this portion of the code directly. I believe it just got shifted or wrapped differently because of the <div> changes else where in the file.
I can try to clean it up though